imports/api/Users/server/methods.js

```diff
@@ -4,6 +4,7 @@ import { Accounts } from 'meteor/accounts-base';
 import editProfile from './edit-profile';
 import exportData from './export-data';
 import deleteAccount from './delete-account';
+import sendWelcomeEmail from './/send-welcome-email';
```

Comment:
There is a typo here. It should be `import sendWelcomeEmail from './send-welcome-email';`
Just remove one of the two slashes